demo/minimalist.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="icon" type="image/png" href="./favicon.png" />
+    <link rel="stylesheet" href="../build/o_spreadsheet.css" />
+    <link rel="stylesheet" href="../node_modules/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="../node_modules/font-awesome/css/font-awesome.min.css" />
+    <title>O-spreadsheet Minimal</title>
+  </head>
+  <body>
+    <script src="../node_modules/@odoo/owl/dist/owl.iife.js"></script>
```

Comment:
I think we should also display a comprehensive list of the runtime dependencies and their use